include/up-cpp/datamodel/validator/UUri.h

```diff
@@ -64,13 +63,13 @@ using ValidationResult = std::tuple<bool, std::optional<Reason>>;
 
 /// @brief Checks if UUri is valid for invoking an RPC method
 ///
-/// The UUri must not be blank/reserved, no field can be a wildcard, and
```

Comment:
Reading between the lines in the spec, reserved values are still a thing. The comments in this header should be restored.

Reply:
To clarify, if the ranges are combined, there are still some values that are reserved or disallowed. However, the new spec is incredibly vague and provides none of the guidance that was available before. I think we may need to file a bug to get the spec cleaned up then we can revisit.